client/.eslintrc.json

```diff
@@ -24,6 +24,8 @@
         "no-param-reassign": [
             "error",
             { "props": true, "ignorePropertyModificationsForRegex": ["^draft"] }
-          ]
+          ],
+          "no-param-reassign": 0
+          
     }
 }
```

Comment:
Add this rule for function decleration.
```js
        "react/function-component-definition": {
            "namedComponents": "function-declaration"
        }
```
And why there is two values for  `no-param-reassign`?